lib/commands/appmanager-livesync.ts

```diff
@@ -0,0 +1,43 @@
+﻿///<reference path="../.d.ts"/>
+"use strict";
+
+export class AppManagerLiveSyncCommand implements ICommand {
+	constructor(private $prompter: IPrompter,
+		private $mobileHelper: Mobile.IMobileHelper,
+		private $appManagerService: IAppManagerService,
+		private $errors: IErrors,
+		private $logger:ILogger) { }
+
+	public execute(args: string[]): IFuture<void> {
+		return ((): void => {
+			if(!args || args.length === 0) {
+				var shouldLivePatchAllPlatforms = this.$prompter.confirm("This command will publish a new update version to AppManager. Are you sure you want to continue?",() => true).wait();
```

Comment:
This is inconsistent. Why does 'appmanager livesync' publish to all platforms, but 'build' errs out with requirement for explicit platform? If we find livesync's behavior to be more user-friendly, maybe in another PR we should change other commands as well?